.github/workflows/CI.yaml

```diff
@@ -204,14 +204,22 @@ jobs:
       matrix:
         helm: [true, false]
```

Comment:
if we must use so many true-false values, I would vote to change them to string constants so that the action name will show human readable settings and not these giant true-false mixes😿 

![image](https://github.com/keptn/lifecycle-toolkit/assets/89971034/85256c30-a3e8-49d3-af31-83c3cb76b24f)

Reply:
adapted